### PR TITLE
Build/bump react and chakra-ui version

### PR DESCRIPTION
**Pull request by Ginakira**
…ersion from 1.8.8 to 2.5.1

### CODE REVIEW

**Comment by johanvx** [comment]
I've also got some version bumps in my local branch. I'll check it this evening and let you know if we can add/change something here.

File: src/lib/theme.js [hunk 1]
```diff
@@ -69,8 +69,7 @@ const fonts = {
 };
 
 const config = {
-  initialColorMode: "light",
-  useSystemColorMode: true
+  initialColorMode: "system"
```
Comment (johanvx) [inline src/lib/theme.js]:
```suggestion
  initialColorMode: "system",
  useSystemColorMode: true
```

I think we can just refer to [the first example in Chakra UI documentation](https://chakra-ui.com/docs/styled-system/color-mode#common-configurations), that is, our UI color mode changes with the system color mode. What do you think?

**Comment by Ginakira** [comment]
The commit message has wrong react version, it should be 18.2.0 instead of 17.2.0. Please correct it when merging.

**Review by johanvx — CHANGES_REQUESTED** [review]

File: src/lib/theme.js (hunk 1 above)
Comment (Ginakira) [reply src/lib/theme.js]: FYI. [migration document](https://chakra-ui.com/getting-started/migration#breaking-changes)

Comment (Ginakira) [reply src/lib/theme.js]: Sorry, you're right. I misunderstood the meaning of the document.

Comment (johanvx) [reply src/lib/theme.js]: Thanks for your information. I understood the related breaking change with color mode.

What I mean (and also the documentation means) is that our App's UI color mode *changes* with the system color mode.

---

[The *first example* in Chakra UI documentation](https://chakra-ui.com/docs/styled-system/color-mode#common-configurations):

```javascript
// A.
// System sets initial value.
// App subscribes to system color mode changes.
const config: ThemeConfig = {
  initialColorMode: 'system',
  useSystemColorMode: true,
}
```

---

**My understanding with a scene:**

It was 17:59 and the system color mode was `light`. Alice opened our App, whose color mode was also `light` as  expected.

Then, it was 18:00 and the system color mode changed to `dark`. Our App:

1. without `useSystemColorMode: true` (as the default value is `false`), our App's UI color mode remained `light`;
2. with `useSystemCOlorMode: true`, our App's UI color mode also changed to `dark`.

Comment (johanvx) [reply src/lib/theme.js]: For your information, like what you've done in your commit, the second example in Chakra UI documentation is:

```javascript
// B.
// System sets initial value.
// App color mode is detached from system color mode changes.
const config: ThemeConfig = {
  initialColorMode: 'system',
  useSystemColorMode: false,
}
```

Pay attention to the comments in the above.

Comment (Ginakira) [reply src/lib/theme.js]: Got it. I've patched that, please review again.

**Review by johanvx — APPROVED** [review]

**Pull request merged**